### PR TITLE
Fixed client to updated `map-markers` and `pieplot` API

### PR DESCRIPTION
**Pull request by bobular**
Just getting the maps working again. There is no issue/ticket for this, but this is essential groundwork to finishing off the maps for b57.

Only dfalke-b back-end seems to have the changes so far. Will need to coordinate roll out to qa back end along with this PR being deployed to qa.

### CODE REVIEW

File: src/lib/core/api/DataClient/types.ts [hunk 1]
```diff
@@ -557,16 +567,21 @@ export interface PieplotRequestParams {
   filters: Filter[];
   config: {
     outputEntityId: string;
-    showMissingness: 'TRUE' | 'FALSE';
+    showMissingness:
+      | 'TRUE'
+      | 'FALSE'
+      | 'noVariables'
+      | 'allVariables'
+      | 'strataVariables';
```
Comment (chowington) [inline src/lib/core/api/DataClient/types.ts:575]: Interested in the meaning of true/false vs noVariables/allVariables. Do they mean different things?

Comment (d-callan) [reply src/lib/core/api/DataClient/types.ts:575]: True/false are essentially deprecated values now, but in the interest of not breaking loads of things.... 
The new values are the other three, where the old false has the same behavior as the new 'noVariables' and the old true has the same behavior as 'strataVariables'

**Review by d-callan — APPROVED** [review]

**Review by chowington — APPROVED** [review]

File: src/lib/core/api/DataClient/types.ts (hunk 1 above)
Comment (chowington) [reply src/lib/core/api/DataClient/types.ts:575]: That makes sense! :)

Comment (bobular) [reply src/lib/core/api/DataClient/types.ts:575]: You got it from two sources, must be true!

**Pull request merged**